web/src/lib/components/memory-page/memory-viewer.svelte

```diff
@@ -72,6 +73,19 @@
 	// Progress should be reset when the current memory or asset changes.
 	$: memoryIndex, assetIndex, reset();
 
+	const handleKeyDown = (e: KeyboardEvent) => {
+		if (e.key === 'ArrowRight' && canGoForward) {
+			e.preventDefault();
+			toNext();
+		} else if (e.key === 'ArrowLeft' && canGoBack) {
+			e.preventDefault();
+			toPrevious();
+		} else if (e.key === 'Escape') {
+			e.preventDefault();
```

Comment:
you can add `|preventDefault` to `<svelte:window on:keydown|preventDefault={handleKeyDown} />` so you don't have to repeat that code in this method, right?

Reply:
@klejejs Pointed out that this prevents the default action of for all keys like `CTRL + R` to refresh, etc.